docs/_templates/autosummary/base.rst

```diff
@@ -0,0 +1,13 @@
+{% if objtype == 'property' %}
+:orphan:
+{% endif %}
+
+{{ fullname | escape | underline}}
+
+.. currentmodule:: {{ module }}
+
+{% if objtype == 'property' %}
+property
+{% endif %}
```

Comment:
Is there a reason to add this? It adds a `<p>property</p>` after the `h1` heading and before the definition of the object:

![image](https://github.com/simpeg/simpeg/assets/11541317/e92d73d9-c839-45c5-b083-d282ab2373e0)

Reply:
I was just taking style from numpy, I’ll remove it for our needs.